lib/modelHelpers.py

```diff
@@ -142,9 +140,12 @@ def send_request(
 
     current_request: GPTMessage = {
         "role": "user",
-        "content": [prompt, content],
+        "content": [prompt],
     }
 
+    if text_to_process is not None:
+        current_request["content"].append(text_to_process)
+
```

Comment:
in a future PR I think we should probably consolidate the text that is operated on and the prompt all into one message. I don't think the model needs both as separate messages and it doesn't make as much sense semantically to split them if we are referring to content that is in a different message. ie `please explain this` `foo bar` seems worse than `please explain this: \n\n foo bar` since `this` could be ambiguous with regards to which message. I agree though that system messages should still be split into separate messages.